code/obj/artifacts/artifact_objects/cloner.dm

```diff
@@ -0,0 +1,105 @@
+
+/obj/artifact/cloner
+	name = "artifact cloner"
+	associated_datum = /datum/artifact/cloner
+
+/datum/artifact/cloner
+	associated_object = /obj/artifact/cloner
+	rarity_class = 4
+	min_triggers = 2
+	max_triggers = 2
+	validtypes = list("ancient","martian","wizard","eldritch","precursor")
+	validtriggers = list(/datum/artifact_trigger/carbon_touch,/datum/artifact_trigger/silicon_touch)
+	react_xray = list(15,90,90,11,"HOLLOW")
+	validtypes = list("wizard","eldritch")
+	touch_descriptors = list("You seem to have a little difficulty taking your hand off its surface.")
+	var/mob/living/carbon/human/clone = null
+	var/imprison_time = 0
+	var/evil_delay = 0
+	var/swapSouls = FALSE
+
+	New()
+		..()
+		imprison_time = rand(50,1200)
+		evil_delay = rand(0,imprison_time)
+		swapSouls = pick(TRUE, FALSE)
```

Comment:
```suggestion
		swapSouls = prob(50)
```

Reply:
Oh, yeah, I guess that works. It doesn't really feel any clearer to me, but I don't mind changing it.